NitroxClient/Communication/Packets/Processors/RocketEndLaunchProcessor.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections;
+using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.MonoBehaviours;
+using NitroxClient.Unity.Helper;
+using NitroxModel.Helper;
+using NitroxModel.Logger;
+using NitroxModel_Subnautica.Packets;
+using UnityEngine;
+
+namespace NitroxClient.Communication.Packets.Processors
+{
+    public class RocketEndLaunchProcessor : ClientPacketProcessor<RocketEndLaunch>
+    {
+        public override void Process(RocketEndLaunch packet)
+        {
+            try
+            {
+                GameObject gameObject = NitroxEntity.RequireObjectFrom(packet.Id);
+                LaunchRocket launchRocket = gameObject.RequireComponentInChildren<LaunchRocket>(true);
+
+                Log.InGame("Hope you enjoyed the Nitrox experience :)");
+
+                launchRocket.ReflectionCall("SetLaunchStarted", false, true);
+                PlayerTimeCapsule.main.Submit(null);
+
+                IEnumerator endSequence = (IEnumerator)launchRocket.ReflectionCall("StartEndCinematic", false, false);
+                launchRocket.StartCoroutine(endSequence);
+
+                HandReticle.main.RequestCrosshairHide();
+            }
+            catch (Exception ex)
+            {
+                Log.Error(ex, "An error occured while processing RocketEndLaunch packet");
+                Log.InGame("Error while processing a rocket launch packet :(");
+                throw;
+            }
```

Comment:
I believe we automatically log at top level (same with below)

Reply:
Was just for testing purpose and will be removed in the final state, I needed to have a visual information when an error occurred, instead of having to check log files every game session